package.js

```diff
@@ -694,44 +694,59 @@ class PathToTarkovController {
         for (let i in locations) {
             if (isValidMap(i)) {
                 for (let x in locations[i].base.exits) {
-                    // Remove extracts restrictions
-                    if (this.config.remove_all_exfils_restrictions && locations[i].base.exits[x].Name !== "EXFIL_Train" && !locations[i].base.exits[x].Name.includes("lab") || locations[i].base.exits[x].Name === "lab_Vent") {
-                        if (locations[i].base.exits[x].PassageRequirement !== "None") {
-                            locations[i].base.exits[x].PassageRequirement = "None";
-                        }
-                        if (locations[i].base.exits[x].ExfiltrationType !== "Individual") {
-                            locations[i].base.exits[x].ExfiltrationType = "Individual";
-                        }
-                        if (locations[i].base.exits[x].Id !== '') {
-                            locations[i].base.exits[x].Id = '';
-                        }
-                        if (locations[i].base.exits[x].Count !== 0) {
-                            locations[i].base.exits[x].Count = 0;
-                        }
-                        if (locations[i].base.exits[x].RequirementTip !== '') {
-                            locations[i].base.exits[x].RequirementTip = '';
-                        }
-                        if (locations[i].base.exits[x].RequiredSlot) {
-                            delete locations[i].base.exits[x].RequiredSlot;
-                        }
-                    }
-
-                    // Make all extractions available to extract
-                    if (locations[i].base.exits[x].Name !== "EXFIL_Train") {
+                    if (locations[i].base.exits[x].Name !== "EXFIL_Train" && !locations[i].base.exits[x].Name.includes("lab") || locations[i].base.exits[x].Name === "lab_Vent") {
+                        // Make all extractions available to extract
                         if (locations[i].base.exits[x].Chance !== 100) {
                             locations[i].base.exits[x].Chance = 100;

```

Comment:
this part should be removed too, [see this comment for more infos](https://github.com/guillaumearm/PathToTarkov/pull/2#issuecomment-1063213817)